themes/bootstrap5/js/lightbox.js

```diff
@@ -543,8 +543,6 @@ VuFind.register('lightbox', function Lightbox() {
         _bsModal.show();
         // Set keyboard focus
         setFocusToFirstNode();
-        // Trigger jQuery event for back-compatibility with facets.js:
-        $(_modal).trigger('show.bs.modal');
```

Comment:
Still learning how to navigate this brave new world. Do the JS files in bootstrap3 and bootstrap5 need to be identical?

Reply:
There's a list in build.xml of the Javascript files that are known to be different. This is one of them, but this PR eliminates facets.js from that list -- thus, if facets.js gets out of sync in future, continuous integration will warn us. But for the files with known differences like lightbox.js, we have to be extra careful.